src/server/src/main/docker/configure-persistence.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
It's unclear why this change is needed.

Does it fix something? Or is it incase someone in the future would like to use bash?

If the latter, then it should be removed from the PR.

Reply:
The configure-persistence uses a bashism and doesn't work with some `sh` implementations, such as the Alpine used by the Dockerfile.